commands/auth/2fa/generate.js

```diff
@@ -0,0 +1,23 @@
+const cli = require('heroku-cli-util')
+const co = require('co')
+
+function * run (context, heroku) {
+  let password = yield cli.prompt('Password', {hide: true})
+  let headers = {'Heroku-Password': password}
+  let codes = yield heroku.post('/account/recovery-codes', {headers: headers})
+  cli.log('Recovery codes:')
+  for (var i in codes) cli.log(codes[i])
+}
+
+const cmd = {
+  description: 'generates and replaces recovery codes',
+  needsAuth: true,
+  run: cli.command(co.wrap(run))
+}
+
+module.exports = [
```

Comment:
@RasPhilCo I am not sure how I missed this the first time, but it is not `auth:2fa:generate`, it is `auth:2fa:generate-recovery-codes`.  I personally like the short version better but we should maintain backwards compatibility